lib/video/screenRecording/firefox/firefoxWindowRecorder.js

```diff
@@ -45,6 +45,29 @@ function findRecordingDirectory(baseDir) {
   return directory;
 }
 
+function findRecordingStartTime(baseDir) {
```

Comment:
I don't understand how this can work.  `readdirSync` won't produce files in any particular order; won't you just get a random timestamp with this method?